platforms/org.eclipse.mita.platform.xdk110/1.0.0/xdk110.platform

```diff
@@ -980,6 +980,27 @@ bus named-singleton GPIO {
 	signal digitalOut(pin : GPIOPin, initialValue : bool = false) : bool
 }
 
+export exception EndOfFileException
+
+io named-singleton SDCard {
+	generator "org.eclipse.mita.platform.xdk110.io.SDCardGenerator"
+	sizeInferrer "org.eclipse.mita.platform.xdk110.io.SDCardSizeInferrer"
+
+	signal resumingFile_Text_Read(filePath: string, blockSize: uint32) : string
+	signal appendingFile_Text_Write(filePath: string) : string
+
+	signal rewindingFile_Text_Read(filePath: string, fileSize: uint32) : string
+	signal rewindingFile_Text_Write(filePath: string) : string
+
+	signal resumingFile_Binary_Read(filePath: string, blockSize: uint32) : array<uint8>
```

Comment:
The signals aren't ordered by their first "word" (e.g. _resuming_, _rewinding_) which makes it hard to check them for completeness.